kyo-prelude/shared/src/main/scala/kyo/kernel/ArrowEffect.scala

```diff
@@ -8,97 +8,187 @@ import scala.annotation.nowarn
+                def input = funcionInput
                 def apply(v: O[A], context: Context)(using Safepoint) =
                     v
     end SuspendOps
 
+    /** See [[SuspendOps.apply]] */
     inline def suspend[A]: SuspendOps[A] = SuspendOps(())
 
     @nowarn("msg=anonymous")
     final class SuspendAndMapOps[A](dummy: Unit) extends AnyVal:
+
+        /** Creates a suspended computation that requests a function implementation and transforms its result immediately upon receipt. This
+          * combines the operations of requesting and transforming a function into a single step.
+          *
+          * @param effectTag
+          *   Identifies which arrow effect to request the function from
+          * @param funcionInput
+          *   The input value to be transformed by the function
+          * @param f
+          *   The function to transform the handler's result
+          * @return
+          *   A computation containing the transformed result
+          */
         inline def apply[I[_], O[_], E <: ArrowEffect[I, O], B, S](
-            inline _tag: Tag[E],
-            inline _input: I[A]
+            inline effectTag: Tag[E],
+            inline funcionInput: I[A]
         )(
-            inline _cont: Safepoint ?=> O[A] => B < S
+            inline f: Safepoint ?=> O[A] => B < S
         )(using inline _frame: Frame): B < (S & E) =
             new KyoSuspend[I, O, E, A, B, S & E]:
                 def frame = _frame
-                def tag   = _tag
-                def input = _input
+                def tag   = effectTag
+                def input = funcionInput
                 def apply(v: O[A], context: Context)(using Safepoint) =
                     Safepoint.handle(v)(
-                        suspend = _cont(v),
-                        continue = _cont(v)
+                        suspend = f(v),
+                        continue = f(v)
                     )
     end SuspendAndMapOps
 
+    /** See [[SuspendAndMapOps.apply]] */
     inline def suspendAndMap[A]: SuspendAndMapOps[A] = SuspendAndMapOps(())
 
     object handle:
+
+        /** Handles an arrow effect by providing a handler function implementation.
+          *
+          * @param effectTag
+          *   Identifies which arrow effect to handle
+          * @param v
+          *   The computation requiring the function implementation
+          * @param handle
+          *   The function implementation to provide
+          * @return
+          *   The computation result with the function implementation provided
+          */
         inline def apply[I[_], O[_], E <: ArrowEffect[I, O], A, B, S, S2, S3](
-            inline tag: Tag[E],
+            inline effectTag: Tag[E],
             v: A < (E & S)
         )(
-            inline handle: [C] => (I[C], Safepoint ?=> O[C] => A < (E & S & S2)) => A < (E & S & S2),
-            inline done: A => B < S3 = (v: A) => v,
-            inline accept: [C] => I[C] => Boolean = [C] => (v: I[C]) => true
```

Comment:
The `accept` function is dangerous and shouldn't be available to users and `done` was unused. I've removed them.